src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java

```diff
@@ -257,51 +257,51 @@ private void addModifier(Element member, Content code) {
     /**
      * Adds the deprecated information for the given member.
      *
-     * @param member      the member being documented.
-     * @param contentTree the content tree to which the deprecated information will be added.
+     * @param member  the member being documented.
+     * @param content the content to which the deprecated information will be added.
      */
-    protected void addDeprecatedInfo(Element member, Content contentTree) {
+    protected void addDeprecatedInfo(Element member, Content content) {
         Content output = (new DeprecatedTaglet()).getAllBlockTagOutput(member,
             writer.getTagletWriterInstance(false));
         if (!output.isEmpty()) {
-            contentTree.add(HtmlTree.DIV(HtmlStyle.deprecationBlock, output));
+            content.add(HtmlTree.DIV(HtmlStyle.deprecationBlock, output));
         }
     }
 
     /**
      * Adds the comment for the given member.
      *
-     * @param member   the member being documented.
-     * @param htmlTree the content tree to which the comment will be added.
+     * @param member  the member being documented.
+     * @param content the content to which the comment will be added.
      */
-    protected void addComment(Element member, Content htmlTree) {
+    protected void addComment(Element member, Content content) {
         if (!utils.getFullBody(member).isEmpty()) {
-            writer.addInlineComment(member, htmlTree);
+            writer.addInlineComment(member, content);
         }
     }
 
     /**
      * Add the preview information for the given member.
      *
      * @param member the member being documented.
-     * @param contentTree the content tree to which the preview information will be added.
+     * @param content the content to which the preview information will be added.
      */
-    protected void addPreviewInfo(Element member, Content contentTree) {
-        writer.addPreviewInfo(member, contentTree);
+    protected void addPreviewInfo(Element member, Content content) {
+        writer.addPreviewInfo(member, content);
     }
 
     protected String name(Element member) {
         return utils.getSimpleName(member);
     }
 
     /**
-     * Adds use information to the documentation tree.
+     * Adds use information to the documentation.
```

Comment:
An alternative would be one of
* Adds use information to the given content.
* Adds use information to the specified content.

Reply:
Or, 

```
Adds use information for a list of elements to the specified content.
@param the list of elements
```